posix-configs/excelsior/mainapp.config

```diff
@@ -5,6 +5,7 @@ param set MAV_BROADCAST 1
 param set SDLOG_PRIO_BOOST 3
 dataman start
 navigator start
+sleep 2
```

Comment:
This is only one of many configs - do we need to update all the others or is this a regression?